os_migrate/plugins/module_utils/image.py

```diff
@@ -2,12 +2,18 @@
 __metaclass__ = type
 
 import hashlib
-import openstack
 import os
 
 from ansible_collections.os_migrate.os_migrate.plugins.module_utils \
     import const, exc, reference, resource
 
+try:
```

Comment:
Looks like we can drop the import indirection changes

Reply:
And we need the "mode" line additions from https://github.com/os-migrate/os-migrate/pull/523/files